relay-profiling/src/lib.rs

```diff
@@ -126,15 +127,18 @@ fn minimal_profile_from_json(data: &[u8]) -> Result<MinimalProfile, ProfileError
     serde_json::from_slice(data).map_err(ProfileError::InvalidJson)
 }
 
-pub fn expand_profile(payload: &[u8]) -> Result<(EventId, Vec<u8>), ProfileError> {
+pub fn expand_profile(
+    payload: &[u8],
+    tags: BTreeMap<String, String>,
+) -> Result<(EventId, Vec<u8>), ProfileError> {
     let profile = match minimal_profile_from_json(payload) {
         Ok(profile) => profile,
         Err(err) => return Err(err),
     };
     let processed_payload = match profile.version {
-        Version::V1 => parse_sample_profile(payload),
+        Version::V1 => parse_sample_profile(payload, tags),
         Version::Unknown => match profile.platform.as_str() {
-            "android" => parse_android_profile(payload),
+            "android" => parse_android_profile(payload, tags),
             "cocoa" => parse_cocoa_profile(payload),
```

Comment:
This is an old format from an old SDK and it's not sending the profile and transaction in the same envelope and extracting tags requires the transaction and profile in the same envelope.

So, not useful to add it since we'll never have them together.